src/styles/days.scss

```diff
@@ -0,0 +1,69 @@
+$square-size: 100px;
+$day-gap: 1px;
+$padding: 10px;
+$width-calendar: $padding * 2 + 7 * $square-size + 6 * $day-gap;
+$left: $day-gap + $square-size;
+$days:
+  "mon" 0,
+  "tue" $left * 2,
+  "wed" $left * 3,
+  "thu" $left * 4,
+  "fri" $left * 5,
+  "sat" $left * 6,
+  "sun" $left * 7;
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+
+  padding: $padding;
+  margin: auto;
+  gap: $day-gap;
+  width: $width-calendar;
+
+  &__day {
+    background-color: #eee;
+    border: 1px solid black;
+    border-radius: 5px;
+
+    height: $square-size;
+    width: $square-size;
+
+    box-sizing: border-box;
+  }
+  @for $number from 1 through 31 {
+    &__day:nth-child(#{$number}):before {
+      display: flex;
+      justify-content: center;
+      text-align: center;
+      margin: 25px;
+      content: "#{$number}";
+      font-family: Arial, Helvetica, sans-serif;
+      font-size: 30px;
+    }
+
+    @for $index from 28 to 31 {
+      &__month-length--#{$index} > :nth-child(n + #{$index + 1}) {
+        display: none;
+      }
+    }
+
+    &:nth-child(n + 30) {
+      display: none;
+    }
+
+    &__day:nth-child(#{$number}):hover {
```

Comment:
This code you can move to line 33 , before closing curly brace, because it's a hover effect on &__day, so this way you can just write &:nth-():hover))